.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ["bug", "triage"]
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Seems unnecessary to me. I think the title should already shortly describe this, rest in how to reproduce

Reply:
I think this help reporting as a contemplating part of reporting a bug